common/devdoc/interlocked_hl_requirements.md

```diff
@@ -165,7 +211,22 @@ MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_SetAndWake
 
 **SRS_INTERLOCKED_HL_02_019: [** `InterlockedHL_SetAndWake` shall succeed and return `INTERLOCKED_HL_OK`. **]**
 
-### InterlockedHL_SetAndWake
+### InterlockedHL_SetAndWake64
+```c
+MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_SetAndWake64, int64_t volatile_atomic*, address, int64_t, value)(INTERLOCKED_HL_OK, INTERLOCKED_HL_ERROR);
+```
+
+`InterlockedHL_SetAndWake64` set the value at `address` to `value` and signals the change of value in `address`. This can be commonly used with `InterlockedHL_WaitForValue64` to signal a waiting thread.
+
+**SRS_INTERLOCKED_HL_05_013: [** If address is NULL then InterlockedHL_SetAndWake64 shall fail and return INTERLOCKED_HL_ERROR. **]**
```

Comment:
>InterlockedHL_SetAndWake64 [](http://example.com/codeflow?start=57&length=26)

backticks

Reply:
Here and everywhere.

Anytime a requirement reference a function, variable name, value, literal etc. it should be in backticks

Reply:
Fixed, earlier, was planning to do only InterlockedHL_WaitForValue and InterlockedHL_WaitForNotValue but looking at the Integ tests, most tests are interdependent on 32 for 32 and 64 for 64. So added these last minute.